tests/Console/ConfigurationResolverTest.php

```diff
@@ -344,6 +344,29 @@ public function testResolvePathRelativeB()
         static::assertSame([__DIR__], $resolver->getPath());
     }
 
+    /**
+     * @dataProvider provideEmptyPathCases
+     */
+    public function testRejectInvalidPath(array $paths)
+    {
+        $resolver = $this->createConfigurationResolver(
+            ['path' => $paths],
+            null,
+            \dirname(__DIR__)
+        );
+
+        $this->expectException(InvalidConfigurationException::class);
```

Comment:
Can we test also for the message? That it starts with "Invalid path".

Reply:
Done.